src/components/pages/Cases/CaseTable.js

```diff
@@ -390,7 +398,7 @@ export default function CaseTable(props) {
       bookmarks.forEach(b => {
         if (savedIds.includes(b.case_id)) {
           // This should be an alert
-          console.log('Case already saved to bookmarks');
+          // console.log('Case already saved to bookmarks');
```

Comment:
Is this one needed? Was it being used as an alert to the user before?

Reply:
Let's pin this one for another time - we may want to change this to an alert of some kind before we ship this app.